addons/aws/1.0.1/kubeadm-join-config-v1beta2.yaml

```diff
@@ -0,0 +1,10 @@
+---
+kind: JoinConfiguration
+apiVersion: kubeadm.k8s.io/v1beta2
+metadata:
+  name: kubeadm-join-configuration
+nodeRegistration:
+  name: $NODE_NAME
+  kubeletExtraArgs:
+    node-ip: $PRIVATE_ADDRESS
```

Comment:
this is already set [here](https://github.com/replicatedhq/kURL/blob/e7dc0cda408ed79da1d7f7411f02ddf4ca1368a0/scripts/kustomize/kubeadm/join-orig/kubeadm-join-config-v1beta2.yaml#L9). if its not necessary remove